### PR TITLE
print pdf code 

### PR DESCRIPTION
**Pull request by ziqiang2344**
removed unwanted jar files
inserted jar files for output pdf
inserted code for output pdf at classes/Student.java

### CODE REVIEW

**Review by YouQuan2007 — COMMENTED** [review]
the "test.pdf" in line 19 is created by ourselves?

File: src/sample/classes/Student.java [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+    public static void main(String[] args) throws FileNotFoundException{
+        List list1= new List();
+        
+        //Insert course list at here
+        list1.add("Java");
+        list1.add("Android");
+        list1.add("Kotlin");
+        list1.add("Android");
+        
+        String path = "Desktop/test.pdf";
```
Comment (Michelle-Lohwt) [inline src/sample/classes/Student.java]: try to change the name of the pdf to something more understandable

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
try to change the name of the pdf to something more understandable

**Comment by ziqiang2344** [comment]
> the "test.pdf" in line 19 is created by ourselves?

it's auto generated

**Review by Michelle-Lohwt — APPROVED** [review]

**Review by YouQuan2007 — APPROVED** [review]

**Pull request merged**